server/certstore_other.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022-2023 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !windows
```

Comment:
we have two styles in other files, not sure if we need both here?

Reply:
Do you mean the build directive?  I believe this is required because on non-windows we want `certstore_other.go` to be seen (and implement the "not-implemented" stub).  This is required because `opts.go` is cross-platform.

The build directive on the `_windows.go` files is redundant since Go compiler is looking for `_windows`.  Do you want me to remove those?

Reply:
Like on the pse_windows_test.go file..

```
//go:build windows
// +build windows
```

Reply:
`certstore_other.go` is scoped as a NOT windows directive.  So it should be included on darwin, linux, etc.

Reply:
Yes I know, I mean to we need both version like the example above?

Reply:
That was perhaps transitional between versions of Go we might be using for build?

[+build deprecated](https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives)